src/components/BannerApp/AppBanner.jsx

```diff
@@ -0,0 +1,16 @@
+import './AppBanner.css'
+import logoImage from '../../assets/images/logo.svg'
+export const AppBanner = () => {
+
+    return (
+        <div className='bannerAppContainer'>
```

Comment:
rename className as the name of the file, and also change the folder name it is still "BannerApp"